hassio_aw2m/config.yaml

```diff
@@ -0,0 +1,33 @@
+name: AmbientWeather2MQTT
+version: "1.0.0"
+slug: hassio_aw2m
+description: This package listens for local data from Ambient Weather stations (such as the WS-2902C) and converts the incoming data to MQTT events.
+url: "https://github.com/neilenns/ambientweather2mqtt"
+startup: application
+boot: auto
+ports:
+  8132/tcp: 7000
+options:
+  MQTT_SERVER: ""
+  MQTT_USERNAME: ""
+  MQTT_PASSWORD: ""
+  MQTT_REJECT_UNAUTHORIZED: false
+  STATION_MAC_ADDRESS: ""
+  TZ: "America/Chicago"
+  VERBOSE: false
+schema:
+  MQTT_SERVER: "str"
+  MQTT_USERNAME: "str"
+  MQTT_PASSWORD: "password"
```

Comment:
Instead of doing it this way can the built-in communication services option be used to auto-discover the MQTT info?

https://developers.home-assistant.io/docs/add-ons/communication